From 305cec4a8952141d7190aef63806acff05107a1d Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Fri, 2 Sep 2016 21:10:02 -0700
Subject: [PATCH 14/39] extract_utils: Add flag to disable pinning

 * We might want to disable pinning entirely for an extract run.
 * Set DISABLE_PINNING=1 to turn it off and override everything.

Change-Id: I1be90dda68f0de1c5f5a70c946052d70bfaab7ed
---
 build/tools/extract_utils.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index 8a0b3f28..a4197a53 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -828,7 +828,7 @@ function extract() {
 
         # Check pinned files
         local HASH="${HASHLIST[$i-1]}"
-        if [ ! -z "$HASH" ] && [ "$HASH" != "x" ]; then
+        if [ "$DISABLE_PINNING" != "1" ] && [ ! -z "$HASH" ] && [ "$HASH" != "x" ]; then
             local KEEP=""
             local TMP="$TMP_DIR/$FROM"
             if [ -f "$TMP" ]; then
-- 
2.15.0

